src/bundle/Resources/views/themes/admin/ui/tabs.html.twig

```diff
@@ -0,0 +1,13 @@
+<ul class="nav nav-tabs ez-tabs">
+    {% for tab in tabs %}
+        <li class="nav-item ez-tabs__tab ez-tabs__tab--primary">
+            <a id="ez-tab-label-{{ tab.id }}" href="#ez-tab-{{ tab.id }}" data-toggle="tab" class="nav-link {% if loop.first %}active{% endif %}">
```

Comment:
It seems to be inconsistent to set first tab as active here and active tab-pane based on passed parameter in `/themes/admin/ui/tab.html.twig`.

Reply:
Using `tab.id` in `<a>` `id` seems to be right, but for some reason (wrong reason I guess) we are using `group` in `<a>` `id` in many places:
- https://github.com/ezsystems/ezplatform-admin-ui/blob/ed9fdfe629d107c6e464382beab536daac11df13/src/bundle/Resources/views/themes/admin/ui/tab/location_view.html.twig#L8
- https://github.com/ezsystems/ezplatform-admin-ui/blob/ed9fdfe629d107c6e464382beab536daac11df13/src/bundle/Resources/views/themes/admin/ui/tab/default.html.twig#L6

Reply:
And now, I realised that we have: https://github.com/ezsystems/ezplatform-admin-ui/blob/ed9fdfe629d107c6e464382beab536daac11df13/src/bundle/Resources/views/themes/admin/ui/tab/default.html.twig so something which we try to create here? 
ping @dew326 @webhdx

Reply:
Propose of [/src/bundle/Resources/views/themes/admin/ui/tab/default.html.twig](https://github.com/ezsystems/ezplatform-admin-ui/blob/ed9fdfe629d107c6e464382beab536daac11df13/src/bundle/Resources/views/themes/admin/ui/tab/default.html.twig) is diffrent: it's default template rendered by  `\EzSystems\EzPlatformAdminUi\Component\TabsComponent` rather then something designed to be reusable :wink: